op-program/client/preinterop.go

```diff
@@ -34,5 +35,8 @@ func RunPreInteropProgram(
 	if err != nil {
 		return err
 	}
-	return claim.ValidateClaim(logger, eth.Bytes32(bootInfo.L2Claim), result.OutputRoot)
+	if validateClaim {
```

Comment:
This is now optional on pre-interop so the block-data hint route can skip validation. The alternative is to append the optimistic output root to the block-data hint. Which should be doable; it's the pending output root in the transition state. But it's not necessary as the validation only tells us what we already know.